src/components/GlobalPreferences/CustomLabels/useSelectIdentities.js

```diff
@@ -35,7 +35,7 @@ function useSelectIdentities(identities, filteredIdentities) {
     selected,
   ])
   const handleToggleIdentity = useCallback(
-    address => () =>
+    address =>
```

Comment:
Just to be sure, is this a fix unrelated to the custom list => DataView change?

Reply:
It is related, before this handler was currying the address into the execution but now the function receives the param when executing.

Reply:
Oh right, it was more a `createHandleToggleIdentity` then :D